src/widgets.ts

```diff
@@ -83,6 +85,39 @@ export abstract class Widget<
     return dispatchWidgetPostMessageEvent
   }
 
+  /**
+   * Public method to communicate with the iframe widget about navigation events
+   * Can be called when the host has a 'back' event happen.
+   * This will send a post message event to the iframe widget, which if it's
```

Comment:
This sentence has some issues. _which if it's_ to _which is_, _it can make its changes and then send_ to _it can then navigate and send_. I think that's better but double check...